bentoml/_internal/server/grpc/server.py

```diff
@@ -134,6 +132,9 @@ async def startup(self) -> None:
             await self.servicer.health_servicer.set(
                 service, pb_health.HealthCheckResponse.SERVING  # type: ignore (no types available)
             )
+        # Running on_startup callback.
+        await self.servicer.startup()
```

Comment:
Do we want to do startup before or after the user services are added? I feel like we still want to run startup code before user services can be called.

Reply:
I think we should run before, but that means the readyz will also be called before adding user service.

Reply:
Sure? I don't think we need to put those services up beforehand.

But maybe it's more correct to move the wait outside of on_startup anyway.

Reply:
maybe the wait can just live here instead of on startup?

Reply:
I think we can close this, and then move the readiness from runner to here?